packages/devtools_app/lib/src/debugger/codeview.dart

```diff
@@ -634,7 +634,7 @@ class LineItem extends StatefulWidget {
 
   static const _hoverDelay = Duration(milliseconds: 150);
   static const _removeDelay = Duration(milliseconds: 50);
-  static const _hoverWidth = 400.0;
+  static double get _hoverWidth => scaleByFontFactor(400.0);
```

Comment:
widths like this all should be claled because they are based on ensuring the size is large enough to fit a reasonable amount of text.